scripts/run_tracker.py

```diff
@@ -205,16 +361,16 @@ def run_bcl2fastq(run, config):
         # Processing cl_options
         threads = cl_options.get('loading-threads')
         if threads and type(threads) is int:
-            cl.extend(['--loading-threads', threads])
+            cl.extend(['--loading-threads', '{}'.format(threads)])
```

Comment:
@guillermo-carrasco  is not '{}'.format(threads) totally equivalent to  '{}'.format(str(threads))?